contrib/dracut/90zfs/module-setup.sh.in

```diff
@@ -61,9 +61,9 @@ install() {
 		dracut_install /usr/lib*/gcc/**/libgcc_s.so*
 	fi
 	# shellcheck disable=SC2050
-	if [ "@LIBFETCH_DYNAMIC@" != 0 ]; then
```

Comment:
Good point, I did not know that the `@@` tags are already quoted. The issue is not solved yet, though.

It is possible to compile OpenZFS without libfetch/libcurl installed and not get a single error/warning. `@LIBFETCH_DYNAMIC@` is an empty string then, leading to
```
if [ != 0 ]; then
```
which is, obviously, syntactically incorrect (that's why SC complained). Now, we'd need either a hard stop at `configure` stage when libfetch/libcurl is not found or handle this oddity in yet different way, e. g.
```
tmp=@LIBFETCH_DYNAMIC@
if [ "$tmp" != 0 ]; then
```

Reply:
That's not something I'd considered! The better solution would be to substitute `@LIBFETCH_DYNAMIC@` without the quotes – it's obviously unexpected, and, as you demonstrate, leads to problems. Cf. #12922.